internal/service/domain/sensor/sensor.go

```diff
@@ -100,6 +106,46 @@ func (s *SensorService) Delete(ctx context.Context, id string) error {
 	return nil
 }
 
+func (s *SensorService) RunStatusUpdater(ctx context.Context, interval time.Duration) {
+	ticker := time.NewTicker(interval)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
+			slog.Info("EVERY two min")
```

Comment:
Please change the text to something meaningful

Reply:
Oh yes, it was just for testing.